pandas/tests/resample/test_base.py

```diff
@@ -26,7 +26,7 @@
 
 # The various methods we support
 downsample_methods = ['min', 'max', 'first', 'last', 'sum', 'mean', 'sem',
-                      'median', 'prod', 'var', 'ohlc', 'quantile']
+                      'median', 'prod', 'var', 'std', 'ohlc', 'quantile']
 upsample_methods = ['count', 'size']
```

Comment:
the other tests that use this are not yet parametrized. what i probably should have done was import the declarations into conftest.py to avoid duplication.

Reply:
ok, if you want to push that change would be good

Reply:
ping on green.